### ISSUE TITLE
Add optional deposit name to config

### ISSUE THREAD

**Issue by Terpo** [oitsjustjose/Geolosys#226]
Forge 106
MC 1.14
Geolosys-1.14.4-4.0.11
---
I've got a more complex deposit:

```
{
      "blocks": [
        "geolosys:coal_ore",
		80,
		"roadstuff:bitumen_ore",
		10,
		"geolosys:kimberlite_ore",
		5,
		"minecraft:coal_block",
		5
      ],
      "samples": [
        "geolosys:coal_ore_sample",
		95,
		"geolosys:kimberlite_ore_sample",
		5
        
      ],
      "yMin": 4,
      "yMax": 54,
      "chance": 8,
      "size": 128,
      "dimBlacklist": [
        "the_nether",
        "the_end"
      ],
      "type": "SPARSE",
      "density": 1.0
    }
```

![grafik](https://user-images.githubusercontent.com/4256058/71447623-fde86200-2730-11ea-83a8-83199fa26131.png)

![grafik](https://user-images.githubusercontent.com/4256058/71447641-35efa500-2731-11ea-9c44-cf3c61916123.png)

Could you add another optional parameter "name" ?
The problem with this would be the translation. Maybe another way would be to generate the name for the field manual from the blocks order or from the spawn chances. 

This are just some ideas and this is by far no big issue ;).
Or simple a list of strings that should be represented ingame in the order given:

 "name": ["geolosys:coal_ore","roadstuff:bitumen_ore","geolosys:kimberlite_ore"],

So I could hide the coal block from the name.

**Comment by oitsjustjose** [comment]
I like this idea, I may implement this as an optional parameter that way if you would rather not use this you don't have to, but if you would you can.

**Comment by oitsjustjose** [comment]
Custom names just got implemented on 1.12, I'll be porting it to 1.14 then 1.15 soon™
